src/app/components/PaperWallet.tsx

```diff
@@ -1,102 +1,216 @@
-    {children}
-    <Body
-      color="BLUE_LIGHT"
-      sx={{
-        position: 'absolute',
-        top: '50%',
-        left: '100%',
-        width: '100%',
-        letterSpacing: '1px',
-        textAlign: 'center',
-        fontWeight: '600',
-        transform: 'translate(-50%, -50%) rotate(-90deg)',
-        textTransform: 'uppercase',
-        fontSize: '13px'
-      }}
+const Block = ({ title, children }: PropsWithChildren<BlockProps>) => (
+  <Flex
+    variant="vertical-start"
+    width="100%"
+    height="100%"
+    backgroundColor="border.dark"
+    sx={{ border: '5px solid', borderColor: 'border.dark', borderRadius: '8px' }}
+  >
+    {title && (
+      <Box pt="9px" pb="14px" width="100%">
+        <Body color="white" lineHeight="20px" fontWeight="bold" textAlign="center">
+          {title}
+        </Body>
+      </Box>
+    )}
+    <Box
+      width="100%"
+      backgroundColor="white"
+      p="7px"
+      sx={{ borderRadius: '6px', boxSizing: 'border-box', flex: '1', textAlign: 'center' }}
     >
-      {title}
-    </Body>
-  </Box>
+      {children}
+    </Box>
+  </Flex>
+);
+
+const NotesBlock = ({ title, children }: PropsWithChildren<BlockProps>) => (
+  <Flex
+    variant="vertical-start"
+    width="100%"
+    height="100%"
+    backgroundColor="BG_GREY_MUTED"
+    sx={{ border: '5px solid', borderColor: 'border.dark', borderRadius: '8px' }}
+  >
+    {title && (
+      <Box
+        pt="9px"
+        pb="14px"
+        width="100%"
+        backgroundColor="white"
+        sx={{
+          borderTopLeftRadius: '6px',
+          borderTopRightRadius: '6px'
+        }}
+      >
+        <Body color="border.dark" lineHeight="20px" textAlign="center">
+          {title}
+        </Body>
+      </Box>
+    )}
+    <Box
+      width="100%"
+      p="7px"
+      sx={{ borderRadius: '6px', boxSizing: 'border-box', flex: '1', textAlign: 'center' }}
+    >
+      {children}
+    </Box>
+  </Flex>
 );
 
+const Line = () => <Box width="100%" height="1px" mt="21px" backgroundColor="BLUE_GREY" />;
+
 interface PaperWalletProps {
+  type: WalletType.PRIVATE_KEY | WalletType.MNEMONIC;
   address: TAddress;
-  mnemonicPhrase: string;
+  secret: string;
 }
 
-export const PaperWallet = forwardRef(({ address, mnemonicPhrase }: PaperWalletProps, ref) => (
-  <Flex
+interface GridBoxProps {
+  last?: boolean;
+  inverted?: boolean;
+}
+
+const GridBox = ({ last, inverted, children }: PropsWithChildren<GridBoxProps>) => (
+  <Box
+    p="3"
+    sx={{
+      boxSizing: 'content-box',
+      borderTop: inverted && '2px dashed',
+      borderRight: !last && !inverted && '2px dashed',
+      borderLeft: !last && inverted && '2px dashed',
+      borderColor: 'border.light',
+      transform: inverted && 'rotate(180deg)'
+    }}
+  >
+    {children}
+  </Box>
+);
+
+export const PaperWallet = forwardRef(({ type, address, secret }: PaperWalletProps, ref) => (
```

Comment:
Should we add the DPath for mnemonic paper wallets?

Reply:
Maybe? That's a good question for design probably.

Reply:
Agreed. I asked them about this.